UA Reference.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2010
+# Visual Studio Version 16
```

Comment:
Is the change to VS2019 by intention?

Reply:
Maybe not, but It should be fine.
As long as  _MinimumVisualStudioVersion = 10.0.40219.1_
Also _UA Core Library.sln_ has _VisualStudioVersion = 16.0.30517.126_

Reply:
luckily vs2017 doesn't mind the version upgrade...